x-pack/plugins/ml/public/application/datavisualizer/stats_datagrid/components/field_data_expanded_row/number_content.tsx

```diff
@@ -0,0 +1,173 @@
+  }, []);
+
+  const defaultChartData: MetricDistributionChartData[] = [];
+  const [distributionChartData, setDistributionChartData] = useState(defaultChartData);
+
+  if (stats === undefined) return null;
+  const { min, median, max, distribution } = stats;
+
+  const summaryTableItems = [
+    {
+      function: 'min',
+      display: (
+        <FormattedMessage
+          id="xpack.ml.fieldDataCardExpandedRow.numberContent.minLabel"
+          defaultMessage="min"
+        />
+      ),
+      value: kibanaFieldFormat(min, fieldFormat),
+    },
+    {
+      function: 'median',
+      display: (
+        <FormattedMessage
+          id="xpack.ml.fieldDataCardExpandedRow.numberContent.medianLabel"
+          defaultMessage="median"
+        />
+      ),
+      value: kibanaFieldFormat(median, fieldFormat),
+    },
+    {
+      function: 'max',
+      display: (
+        <FormattedMessage
+          id="xpack.ml.fieldDataCardExpandedRow.numberContent.maxLabel"
+          defaultMessage="max"
+        />
+      ),
+      value: kibanaFieldFormat(max, fieldFormat),
+    },
+  ];
+  const summaryTableColumns = [
+    {
+      name: '',
+      render: (summaryItem: { display: ReactNode }) => summaryItem.display,
+      width: '75px',
+    },
+    {
+      field: 'value',
+      name: '',
+      render: (v: string) => <strong>{v}</strong>,
+    },
+  ];
+
+  const summaryTableTitle = i18n.translate(
+    'xpack.ml.fieldDataCardExpandedRow.numberContent.summaryTableTitle',
+    {
+      defaultMessage: 'Summary',
+    }
+  );
+  return (
+    <EuiFlexGroup direction={'row'} data-test-subj={'mlNumberSummaryTable'}>
+      <EuiFlexItem>
+        <ExpandedRowFieldHeader>{summaryTableTitle}</ExpandedRowFieldHeader>
+        <EuiBasicTable<SummaryTableItem>
+          className={'mlDataVisualizerSummaryTable'}
+          compressed
+          items={summaryTableItems}
+          columns={summaryTableColumns}
+          tableCaption={summaryTableTitle}
+        />
+      </EuiFlexItem>
+      <VerticalSpacer />
+      {stats && (
+        <EuiFlexItem>
+          <EuiFlexGroup direction={'column'} data-test-subj={'mlTopValues'}>
+            <EuiFlexItem grow={false}>
+              <ExpandedRowFieldHeader>
+                <FormattedMessage
+                  id="xpack.ml.fieldDataCardExpandedRow.numberContent.topValuesTitle"
+                  defaultMessage="Top values"
+                />
+              </ExpandedRowFieldHeader>
+            </EuiFlexItem>
+            <EuiFlexItem>
+              <TopValues
+                stats={stats}
+                fieldFormat={fieldFormat}
+                barColor="primary"
+                compressed={true}
+              />
+            </EuiFlexItem>
+          </EuiFlexGroup>
+        </EuiFlexItem>
+      )}
+      <VerticalSpacer />
+
+      {distribution && (
+        <EuiFlexItem>
+          <EuiFlexGroup direction={'column'} data-test-subj={'mlMetricDistribution'}>
```

Comment:
question: do we need a flex layout here?

Reply:
Updated here https://github.com/elastic/kibana/pull/85726/commits/a021855fbe9e0d6d08dbfd2f6b847f25c3f12a80